src/nvim/event/loop.c

```diff
@@ -172,7 +172,7 @@ bool loop_close(Loop *loop, bool wait)
       log_uv_handles(&loop->uv);
       break;
     }
-#if defined(EXITFREE)
+#if defined(NOT_CLOSE_UV)
```

Comment:
the name is confusing to me. Maybe `EXIT_IMMEDIATELY` or something.

Reply:
I think that's a bit too general. What about `SKIP_UV_CLOSE` or something?